Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/00_illuminant_tex.material

```diff
@@ -1,11 +1,11 @@
 {
     "description": "",
-    "materialType": "Materials\\Types\\StandardPBR.materialtype",
-    "parentMaterial": "Materials\\Presets\\MacBeth\\00_illuminant.material",
-    "propertyLayoutVersion": 3,
+    "parentMaterial": "00_illuminant.material",
+    "materialType": "../../Types/StandardPBR.materialtype",
+    "materialTypeVersion": 4,
     "properties": {
         "baseColor": {
-            "textureMap": "Materials/Presets/MacBeth/00_illuminant_sRGB.tif"
+            "textureMap": "00_illuminant_sRGB.tif"
         }
     }
-}
+}
```

Comment:
no new line at end of file. same with many other files

Reply:
I didn't actually edit any of the files specifically. Everything was done by saving material files in the Material Editor.